c2corg_api/models/report.py

```diff
@@ -0,0 +1,263 @@
+import colander
+from c2corg_api.models.schema_utils import restrict_schema,\
+    get_update_schema, get_create_schema
+from c2corg_common.fields_report import fields_report
+from sqlalchemy import (
+    Column,
+    Integer,
+    Boolean,
+    SmallInteger,
+    ForeignKey,
+    String,
+    Date
+    )
+
+from colanderalchemy import SQLAlchemySchemaNode
+
+from c2corg_api.models import schema, Base
+from c2corg_api.models.utils import ArrayOfEnum
+from c2corg_api.models.utils import copy_attributes
+from c2corg_api.models.document import (
+    ArchiveDocument,
+    Document,
+    geometry_schema_overrides,
+    schema_attributes, DocumentLocale, ArchiveDocumentLocale,
+    schema_locale_attributes)
+from c2corg_common import document_types
+from c2corg_api.models import enums
+
+REPORT_TYPE = document_types.REPORT_TYPE
+
+
+class _ReportMixin(object):
+    # Caractérisation de l'évènement - Event characterization
+    elevation = Column(SmallInteger)  # Altitude
+    date = Column(Date)  # nullable=False date des observations
+    # * # Type d'évènement
+    event_type = Column(ArrayOfEnum(enums.event_type))  # nullable=False
+    activities = Column(ArrayOfEnum(enums.activity_type))  # nullable=False
+
+    # * # Nombre de participants
+    nb_participants = Column(SmallInteger)  # nullable=False
+    nb_impacted = Column(SmallInteger)  # Nombre de personnes touchées
```

Comment:
Sometimes the descriptions in the comments are on the same line and sometimes in the line above. Could you please make it consistent?